libdedx/dedx.c

```diff
@@ -663,6 +663,27 @@ int _dedx_load_atima(stopping_data * data, dedx_config * config, float * energy,
 	return 0;
 }
 
+//energy - in MeV/nucleon, unless  program is ESTAR, then MeV
```

Comment:
add an example code with call to this method, similar as it was done here: https://github.com/APTG/libdedx/pull/34/files